requirements.txt

```diff
@@ -1 +1,3 @@
 rasa==1.10.2
+sentry-sdk==0.15.1
+aiocontextvars==0.2.2
```

Comment:
I don't see us using this anywhere. Is this something that Sentry needs but doesn't depend on?